tests/components/pushbullet/fixtures/chats.json

```diff
@@ -0,0 +1,18 @@
+{
+  "chats": [
+    {
+      "active": true,
+      "created": 1412047948.579029,
+      "iden": "ujpah72o0sjAoRtnM0jc",
+      "modified": 1412047948.579031,
+      "with": {
+        "email": "carmack@idsoftware.com",
+        "email_normalized": "carmack@idsoftware.com",
```

Comment:
Can we use some fake data instead and not reference real alive people?

Reply:
Changed to fake data.